lib/GADS/Schema/Result/Report.pm

```diff
@@ -272,29 +286,34 @@ Function to create a PDF of the report - it will return a PDF object
 sub create_pdf
 {   my ($self, $record) = @_;
 
-    my $dateformat = GADS::Config->instance->dateformat;
-    my $now        = DateTime->now;
-    $now->set_time_zone('Europe/London');
-    my $now_formatted = $now->format_cldr($dateformat) . " at " . $now->hms;
-    my $updated =
-      $self->created->format_cldr($dateformat) . " at " . $self->created->hms;
-
-    my $config = GADS::Config->instance;
-    my $header = $config && $config->gads && $config->gads->{header};
-    my $pdf    = CtrlO::PDF->new(
-        header => $header,
-        footer => "Downloaded by " . $self->user->value . " on $now_formatted",
-    );
-
-    $pdf->add_page;
-    $pdf->heading( $self->name );
-    $pdf->heading( $self->description, size => 14 ) if $self->description;
-
-    my $fields = [ [ 'Field', 'Value' ] ];
+    my $marking = $self->_read_security_marking;
+    my $logo = $self->instance->site->create_temp_logo;
+
+    my $pdf;
+
+    if($logo) {
+        $pdf    = CtrlO::PDF->new(
+            header => $marking,
+            footer => $marking,
+            logo   => $logo,
+        );
+        $pdf->add_page;
+        my $topmargin = -30;
+        $pdf->heading( $self->title || $self->name, topmargin=>$topmargin );
+        $pdf->text($self->description , size => 14 ) if $self->description;
+    } else {
+        $pdf    = CtrlO::PDF->new(
+            header => $marking,
+            footer => $marking,
+        );
+        $pdf->add_page;
+        $pdf->heading( $self->title || $self->name );
+        $pdf->heading($self->description , size => 14 );
```

Comment:
The description heading in the previous block is only set if the description exists, where as it is set in this statement regardless? It might be worth combining these statements outside of the `if` statement to make them consistent? (and maybe others such as `add_page`)